puppet/zulip_ops/files/needrestart/zulip.conf

```diff
@@ -0,0 +1,11 @@
+# -*-cperl-*-
+my @ignore = (
+    qr/^memcached\.service$/,
+    qr/^nginx\.service$/,
+    qr/^postgresql(@\d+)?.service$/,
```

Comment:
`postgresql@14-main.service` won’t match this.

Reply:
Thanks for the catch; updated to be more generic.